person.rb

```diff
@@ -2,17 +2,18 @@
 require_relative 'decorator'
 
 class Person < Nameable
+  attr_accessor :name, :age
+  attr_reader :id, :rentals
+
   def initialize(id, age, name = 'Unknown', parent_permission: true)
     @id = id
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
     super()
   end
 
-  attr_accessor :name, :age
-  attr_reader :id
-
   def can_use_services?
     of_age? || @parent_permission
   end
```

Comment:
- [ ]  You should have add_rental method for adding both book and date. take a reference [here ](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship)for more information